locales/zh-CN.json

```
@@ -22,8 +22,8 @@
     "decreasing": "递减型",
     "fluctuating": "波动型",
     "unknown": "不知道",
-    "large-spike": "大幅上涨（三期型）",
-    "small-spike": "小幅上涨（四期型）"
+    "large-spike": "大幅上涨",
+    "small-spike": "小幅上涨"
```

```suggestion
    "large-spike": "三期型",
    "small-spike": "四期型"
```
In fact, I think it would be better to use `三期型` and `四期型` (Remove `大幅上涨` and `小幅上涨`).

---

无法理解啥是三期型四期型，可以麻烦解释一下？对于普通人来说，跟我一样有疑问的会有更多

---

> 无法理解啥是三期型四期型，可以麻烦解释一下？对于普通人来说，跟我一样有疑问的会有更多

简单来说，三期型上涨三次，四期型上涨四次。
https://github.com/mikebryant/ac-nh-turnip-prices/pull/166

---

那么我觉得可以翻译成这样，而不是简单的三期型和四期型

三期型（上涨三次）
四期型（上涨四次）

---

不经过解释的话，很多人都不会知道这两词的意思

---

因此在最初翻译的时候我就考虑了很久，我也并不确定哪种翻译才是最佳的。

---

三期型和四期型最开始再繁体的资料里面看到。如果当做资料来看的话，会有更多解释来支撑这个翻译，而这个网站一般针对更多的是普通玩家，不建议沿用

---

已修正为，让更多人容易理解

三期型（上涨三次）
四期型（上涨四次）

---

My worry about "up three times" or "up four times" is they don't explain that one is better than the other. People will need to learn what they mean from somewhere else, which is not great.

Three large increases could be better than four small increases.

A way to explain that one has a large increase, and the other has a small increase seems easier for regular people to understand.

That is why in English we chose "large spike" and "small spike"

---

It maybe better? It's too difficult

三期型（大幅上涨三次）
四期型（小幅上涨四次）

---

大幅上涨三次
小幅上涨四次

---

> My worry about "up three times" or "up four times" is they don't explain that one is better than the other. People will need to learn what they mean from somewhere else, which is not great.
> 
> Three large increases could be better than four small increases.
> 
> A way to explain that one has a large increase, and the other has a small increase seems easier for regular people to understand.
> 
> That is why in English we chose "large spike" and "small spike"

The concept of large and small is not clear. Perhaps it would be more intuitive to let  users judge by the number of rises.

---

> It maybe better? It's too difficult
> 
> 大幅上涨三次
> 小幅上涨四次

三期型（大幅上涨三次）
四期型（小幅上涨四次）

---

> > It maybe better? It's too difficult
> > 大幅上涨三次
> > 小幅上涨四次
> 
> 三期型（大幅上涨三次）
> 四期型（小幅上涨四次）

It's too many word

---

Perhaps it would be better to explain the patterns by adding title attribute.

---

I think 大幅上涨三次 and 小幅上涨四次 is enough to explain, and it seems easier for regular people to understand.

---

There is an issue open #41 for adding tooltip/hover that explains how the pattern works.

The idea behind the title is to give a simple name that explains that overall result of the pattern, not each phase of that pattern.

For example, "large spike" means the overall result is a big increase in sell price, but doesn't indicate how many periods the increase will last.